site/quickstart.md

```diff
@@ -0,0 +1,262 @@
+JAVA_PKG=jdk-8u201-linux-x64.tar.gz --build-arg PATCHDIR=patches /home/acmeuser/wlsimgbuilder_temp8791654163579491583 
+Sending build context to Docker daemon   1.08GB
+
+Step 1/46 : ARG BASE_IMAGE=oraclelinux:7-slim
+...
+Removing intermediate container 57ccb9fff56b
+ ---> 18d366fc3da4
+Successfully built 18d366fc3da4
+Successfully tagged wls:12.2.1.3.0
+```
+
+Once the image is created, you can use any docker commands on the image, for example:
+
+```bash
+docker images
+
+REPOSITORY          TAG                 IMAGE ID            CREATED              SIZE
+wls                 12.2.1.3.0          18d366fc3da4        About a minute ago   1.41GB
+oraclelinux         7-slim              f7512ac13c1b        6 weeks ago          118MB
+
+```
+
+## Creating image with no internet access
+
+
+In this use case, since there is no internet access.  You will be responsible for downloading all the installers and 
+patches, plus setting up the cache.  You also have to provide a base operating system image that has the following packages 
+installed.
+
+
+```bash
+gzip 
+tar 
+unzip
+```
+
+For each WebLogic patches, you will need to download it from Oracle Support and set up the cache. For example, if you downloaded patch number 
+27342434 for WebLogic version 12.2.1.3.0:
+ 
+```bash
+imagetool cache addPatch --patchId 27342434 --version 12.2.1.3.0 --path /home/acmeuser/cache/p27342434_122130_Generic .zip -user username@mycompany.com --passwordEnv MYPWD
+```
+
+You need to provide the credential in the command line, it verifies the MD5 on the file system against the meta data on 
+Oracle Support for this patch number
+
+Then, you can run the command to create the image:
+
+```bash
+imagetool create --fromImage myosimg:latest --tag wls:12.2.1.3.0 --patches 27342434 --version 12.2.1.3.0 --useCache always
+```
+
+Sometimes, a WebLogic patch may require patching the OPatch binaries before applying them.  We recommend 
+downloading the latest OPatch's patch and setup the cache.  For example, the latest OPatch's patch is 28186730, 13.9
+.4.0.0.  You can use this command to setup the cache after downloading from Oracle Support.
+
+```bash
+imagetool cache addPatch --patchId 28186730 --version 13.9.4.0.0 --path /home/acmeuser/cache/p28186730_139400_Generic.zip
+```
+
+## Patching an existing image
+
+This use case allows you apply WebLogic patches to an existing image. It works similar to previous use cases, you 
+have the option to download automatically by the tool or manual downloading the patches yourself.
+
+Once the cache is setup, you can use the following command to update an image:
+
+```bash
+imagetool update --fromImage wls:12.2.1.3.0 --tag wls:12.2.1.3.4 --patches 27342434 --version 12.2.1.3.0 --useCache always
+```
+
+## Create an image with a WebLogic Domain using WebLogic Deploy Tool
+
+The image tool allows create a customized WebLogic domain using [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling).  
```

Comment:
The image tool allows create a customized -> The Image Tool allows creating a customized